Apps/WebClient/src/ClientApp/src/components/NotificationCentreComponent.vue

```diff
@@ -148,4 +168,8 @@ export default class NotificationCentreComponent extends Vue {
 
 <style lang="scss" scoped>
 @import "@/assets/scss/_variables.scss";
+
+.noNotification {
+    min-height: 75vh;
+}
```

Comment:
We should use d-flex and flex-grow-1 to take up all the space allotted to the parent instead of relying on viewport height.